src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResource.kt

```diff
@@ -261,4 +262,64 @@ class VisitResource(private val visitService: VisitService) {
         ignoreMissingRoom = ignoreMissingRoom
       )
     )
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_VISITS')")
+  @GetMapping("/visits/rooms/usage-count")
+  @ResponseStatus(HttpStatus.OK)
```

Comment:
don't think you need to specify response status if it is the default `OK` response

Reply:
removed